Automattic/vip-go-mu-plugins
# Adds command to output SQL for conversion.

## Description

This will allow us to output SQL for site conversions, rather than doing the conversion live.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out PR.
1. Go to shell/sandbox
1. Find site with non-utf8mb4 tables
1. Run `wp vip utf8mb4 create-sql`
1. Verify SQL looks correct
